os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_central_client_app.c

```diff
@@ -358,6 +363,16 @@ int ble_tizenrt_central_handle_upstream_msg(uint16_t subtype, void *pdata)
             le_bond_clear_all_keys();
         }
 			break;
+        case BLE_TIZENRT_MODIFY_WHITELIST:
+        {
+            debug_print("\r\n[%s] le_modify_white_list", __FUNCTION__);
+            T_TIZENRT_MODI_WHITELIST_PARAM *param = pdata;
+            ret = le_modify_white_list(param->type, param->remote_bd, param->remote_bd_type);
+            if(ret)
+                printf("\r\n[%s] 0x%x le_modify_white_list fail  ", __FUNCTION__, ret);
+            os_mem_free(param);
```

Comment:
I see some cases param/data is not freed, is it possible to put at the end of function to clear the memory